src/index.js

```diff
@@ -51,19 +54,68 @@ var handlers = {
     'ReadContentAtPositionIntent': readContentAtPosition,
 
     'MoreIntent': function() {
-        // repeat last intent action with increased offSet
-        this.emit(this.event.session.attributes.lastIntent, false);
+        const lastIntent = this.event.session.attributes.lastIntent;
```

Comment:
Nitpicking, but here you are:

1. creating a `const` reference to an object
2. modifying the object using a _different_ reference
3. returning the _same_ object modified using the _other_ `const` reference 

I find the use of `const` here rather misleading 😅 I would just use `this.event.session.lastIntent` everywhere.